lib/index.js

```diff
@@ -149,7 +151,7 @@ function getOptions(options) {
 
   options.error = function(err) {
     if (error) {
-      error(util._extend(new Error(), JSON.parse(err)));
+      error.call(options.context, util._extend(new Error(), JSON.parse(err)));
```

Comment:
Oh nice! I thought it will drop those. Thanks for the example! (we can add this test to the mix as well) :+1: 

Separately, are you suggesting if we drop `options.context`, then any new member to `options` set by node-sass will be lost in callbacks? Can you please provide a concrete example for [this](https://github.com/sass/node-sass/pull/686#discussion_r24995696), which shows that it will fail on some account in case we drop `.call(options.context, ...)` in favour of `.call(options, ...)`?